docs/unified-test-documentation/dasharo-compatibility/340-docking-station-net-interface.md

```diff
@@ -0,0 +1,102 @@
+# Dasharo Compatibility: Docking station NET interface
```

Comment:
It would be good to actualize test matrixes for all NC laptops

Reply:
Done